datalad/utils.py

```diff
@@ -1101,10 +1101,41 @@ def get_path_prefix(path, pwd=None):
         return path
 
 
+def _get_normalized_paths(path, prefix):
+    if isabs(path) != isabs(prefix):
+        raise ValueError("Bot paths must either be absolute or relative. "
+                         "Got %r and %r" % (path, prefix))
```

Comment:
I wonder if any test starts to fail with this... pointing to a potential problem in the code since we should have then resolved paths to full before comparing for commonprefix

Reply:
ok, according to he coverage -- not covered which is good, i.e. was not hit, but we need coverage for it, so will do now